flask_common/mongo/fields/crypto.py

```diff
@@ -48,10 +48,14 @@ def _decrypt(self, data):
         raise AuthenticationError('message authentication failed')
 
     def to_python(self, value):
-        return value and self._decrypt(value) or None
+        if not value:
+            return None
+        return self._decrypt(value)
```

Comment:
I think @tsx was suggesting something like
`return self._decrypt(value) if value else None`

Reply:
Oops, you're right! Done in https://github.com/closeio/flask-common/pull/71/commits/96a4def567e3725bd0858f1b2ce02c8501db9791.